.idea/libraries/buck_lib.xml

```diff
@@ -8,7 +8,6 @@
       <root url="jar://$PROJECT_DIR$/third-party/java/jna/jna-4.4.0.jar!/" />
```

Comment:
Please, update this too

Reply:
This is still pointing to the old version

Reply:
I didn’t realize pull requests needed to update IDE files.  I’ll be happy to update it though.